src/intentManager.js

```diff
@@ -305,6 +305,18 @@ define([
         }
     });
 
+    function noIntents(features) {
+        return !features.custom_intents && !features.templated_intents;
+    }
+
+    function templatedIntents(features) {
+        return features.templated_intents && !features.custom_intents;
```

Comment:
er - see they are both true in tests - is the idea that if `custom` is true then you don't show these even if they're on?